apps/algorithm/search/src/test_server.py

```diff
@@ -1,2 +1,1323 @@
-def test_addition():
-    assert 1 + 2 == 3
+import os
+
+import pytest
+from elasticsearch import Elasticsearch
+from fastapi.testclient import TestClient
+
+from server import app
+
+TEST_INDEX = "test-index"
+
+client = TestClient(app)
+
+es_endpoint = os.getenv("ES_ENDPOINT")
+es = Elasticsearch([es_endpoint], verify_certs=False)
+
+
+@pytest.fixture(scope="function", autouse=True)
+def setup_and_teardown_index(monkeypatch):
+    monkeypatch.setenv("ES_INDEX", TEST_INDEX)
```

Comment:
We talked about using a different "random" Index for each test for isolation. If we're switching to async tests (for postgres connections), this'll be required

Reply:
I'll do that in a PR when I add the postgres stuff and switch to async tests.